scripts/postinstall.js

```diff
@@ -11,8 +12,40 @@ var directories = {
 };
 
 console.log('NativeScript Firebase Plugin Installation');
-prompt.start();
-askiOSPrompt();
+
+var appRoot = require('app-root-path').toString();
+var pluginConfigPath = path.join(appRoot, "firebase.nativescript.json");
+
+var config = {};
+function mergeConfig(result) {
+    for (var key in result) {
+        config[key] = isSelected(result[key]);
+    }
+}
+function saveConfig() {
+    fs.writeFileSync(pluginConfigPath, JSON.stringify(config, null, 4));
+}
+function readConfig() {
+    try {
+        config = JSON.parse(fs.readFileSync(pluginConfigPath));
+    } catch(e) {
+        console.log("Failed reading config at " + pluginConfigPath);
+        console.log(e);
+        config = {};
+    }
+}
+
+if (process.argv.indexOf("config") == -1 && fs.existsSync(pluginConfigPath)) {
```

Comment:
No, there are now two npm scripts in the plugin now. One of them is the postinstall that will check for configuration json and does the configuration. The other is "config" which is run using `npm run config` in the plugin folder and provides the "config" arg to the same script the post install is running. It will force the script to ignore the configuration.json and re-ask the configuration questions.